packages/admin-ui/src/visually-hidden/index.tsx

```diff
@@ -0,0 +1,11 @@
+import { createComponent, useElement } from '@vtex/admin-ui-react'
```

Comment:
index files only export things. Create a new `visually-hidden.ts` and export from it.